mlapi/discriminating_algo.py

```diff
@@ -0,0 +1,129 @@
+import statistics
+
+class DiscriminatingAlgo(object):
+    # Discrimination parameters
+    facets_sample_count = 3  # minimum docs by facet to be accepted
+    tolerance_interval = 10 # max standard deviation accepted ()
+    nbr_facet_values = 2 # to consider a facet as having a large number of values
+
+    facets = [] # names
+    step2 = {}
+    step3 = {}
+
+    def discriminating_facets_algo(self, facets_by_document):
```

Comment:
We usually name methods with an action verb. What does this return? Are there side-effect (I think not)?. Put that info in the method name.

Reply:
corrigé